Vuforia Spatial Toolbox/WebView/REWebView.mm

```diff
@@ -172,4 +172,19 @@ - (void)clearCache
                                            completionHandler:^{}];
 }
 
+- (void)setAspectRatio:(float)ratio
+{
+    CGRect newFrame = self.frame;
+    if (ratio == 0) {
+        newFrame.size.width = [UIScreen mainScreen].bounds.size.width;
+        newFrame.size.height = [UIScreen mainScreen].bounds.size.height;
+    } else {
+        newFrame.size.width = [UIScreen mainScreen].bounds.size.height * ratio;
```

Comment:
It would be a bit more future-proof to also handle cases where we're doing something like running on an iPad and (ratio * height) > width

Reply:
Good catch! Just fixed that.